src/index.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
+import './App.css';
+import Calculator from './components/Calculator';
 import reportWebVitals from './reportWebVitals';
 import 'bootstrap/dist/css/bootstrap.css';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
```

Comment:
- [ ] Dear @Katended, You did well making the most of the changes but, as suggested by the previous reviewer, your App.js(main component) should have the imported Calculator.js component and the App should be used here in index.js as it is generated by React.

Reply:
@TadesseDev I am not sure but I seem to find this requested change already existing in the codebase. I have mae a [commit at](https://github.com/Katended/math-magicians/commit/ef684d462ad4514eb938b72d64c3c915e1f8bac4) . And it seems to suggest so.

Reply:
- dear @Katended, It seems like you pushed commits while I was reviewing (I can also see you are still pushing commits), which are not part of my reviews. Usually, when you submit a review you should have added all the changes. Please verify all the changes are in place and ask for a re-review.